keyboards/dyz/synthesis60/info.json

```diff
@@ -59,85 +59,164 @@
+                {"label": "]", "matrix": [1, 12], "x": 15.75, "y": 1},
+                {"label": "\\", "matrix": [1, 13], "x": 16.75, "y": 1, "w": 1.5},
+
+                {"label": "Page Down", "matrix": [4, 2], "x": 0, "y": 2},
+
+                {"label": "Caps Lock", "matrix": [2, 0], "x": 1.25, "y": 2, "w": 1.75},
+                {"label": "A", "matrix": [2, 1], "x": 3, "y": 2},
+                {"label": "S", "matrix": [2, 2], "x": 4, "y": 2},
+                {"label": "D", "matrix": [2, 3], "x": 5, "y": 2},
+                {"label": "F", "matrix": [2, 4], "x": 6, "y": 2},
+                {"label": "G", "matrix": [2, 5], "x": 7, "y": 2},
+
+                {"label": "H", "matrix": [2, 6], "x": 10, "y": 2},
+                {"label": "J", "matrix": [2, 7], "x": 11, "y": 2},
+                {"label": "K", "matrix": [2, 8], "x": 12, "y": 2},
+                {"label": "L", "matrix": [2, 9], "x": 13, "y": 2},
+                {"label": ";", "matrix": [2, 10], "x": 14, "y": 2},
+                {"label": "'", "matrix": [2, 11], "x": 15, "y": 2},
+                {"label": "Enter", "matrix": [2, 12], "x": 16, "y": 2, "w": 2.25},
+
+                {"label": "Shift", "matrix": [3, 0], "x": 1.25, "y": 3, "w": 2.25},
+                {"label": "Z", "matrix": [3, 1], "x": 3.5, "y": 3},
+                {"label": "X", "matrix": [3, 2], "x": 4.5, "y": 3},
+                {"label": "C", "matrix": [3, 3], "x": 5.5, "y": 3},
+                {"label": "V", "matrix": [3, 4], "x": 6.5, "y": 3},
+                {"label": "B", "matrix": [3, 5], "x": 7.5, "y": 3},
+
+                {"label": "B", "matrix": [3, 6], "x": 9.5, "y": 3},
+                {"label": "N", "matrix": [3, 7], "x": 10.5, "y": 3},
+                {"label": "M", "matrix": [3, 8], "x": 11.5, "y": 3},
+                {"label": ",", "matrix": [3, 9], "x": 12.5, "y": 3},
+                {"label": ".", "matrix": [3, 10], "x": 13.5, "y": 3},
+                {"label": "/", "matrix": [3, 11], "x": 14.5, "y": 3},
+                {"label": "Shift", "matrix": [3, 12], "x": 15.5, "y": 3, "w": 1.75},
+                {"label": "Delete", "matrix": [3, 13], "x": 17.25, "y": 3},
+
+                {"label": "Ctrl", "matrix": [4, 3], "x": 1.25, "y": 4, "w": 1.5},
+                {"label": "Alt", "matrix": [4, 4], "x": 4, "y": 4, "w": 1.5},
+                {"label": "Space", "matrix": [4, 5], "x": 5.5, "y": 4, "w": 2.25},
+                {"label": "Fn", "matrix": [4, 6], "x": 7.75, "y": 4},
+
+                {"label": "Space", "matrix": [4, 7], "x": 9.5, "y": 4, "w": 1.5},
+                {"label": "Space", "matrix": [4, 8], "x": 11, "y": 4, "w": 1.25},
+                {"label": "Alt", "matrix": [4, 9], "x": 12.25, "y": 4, "w": 1.5},
+                {"label": "GUI", "matrix": [4, 10], "x": 14.5, "y": 4},
+                {"label": "Menu", "matrix": [4, 11], "x": 15.5, "y": 4},
+                {"label": "Ctrl", "matrix": [4, 12], "x": 16.5, "y": 4, "w": 1.5}
+            ]
+        },
+
```

Comment:
```suggestion
```

Reply:
Is there a specific reason for this nitpick?

Reply:
It's formatting that will be blown away by `qmk format-json`. I have a PR in the works to clean up some whitespace issues across all info.jsons (tabs to spaces, trailing whitespace, newline at EOF, etc).

Technically, the line breaks in the layouts would also be removed; I've thought about trying to automate it by inspecting the previous key location and size. Maybe it's time better spent on writing a GUI tool for building layouts instead.